tests/checkpoint/test_checkpoint.py

```diff
@@ -224,111 +225,6 @@ def test_checkpoint_serialization(
         self._assert_valid_uuid(actual["validation_definitions"][0]["id"])
         self._assert_valid_uuid(actual["validation_definitions"][1]["id"])
 
-    @pytest.mark.filesystem
-    def test_checkpoint_filesystem_round_trip_adds_ids(
```

Comment:
Round trip should not add ids - we will inform the user of the issue and ask them to persist each item one by one